modules/swagger-jaxrs/src/main/java/com/wordnik/swagger/jaxrs/Reader.java

```diff
@@ -53,16 +61,25 @@
 import com.wordnik.swagger.models.properties.MapProperty;
 import com.wordnik.swagger.models.properties.Property;
 import com.wordnik.swagger.models.properties.RefProperty;
-import com.wordnik.swagger.util.Json;
 
 public class Reader {
   private static final Logger LOGGER = LoggerFactory.getLogger(Reader.class);
   private static final String SUCCESSFUL_OPERATION = "successful operation";
   private static final String PATH_DELIMITER = "/";
-
+  private static final Set<Class<? extends Annotation>> FIELD_ANNOTATIONS;
   Swagger swagger;
   private final ReaderConfig config;
 
+  static {
+    final Set<Class<? extends Annotation>> fieldAnnotations = new HashSet<Class<? extends Annotation>>();
+    fieldAnnotations.add(PathParam.class);
+    fieldAnnotations.add(QueryParam.class);
+    fieldAnnotations.add(HeaderParam.class);
+    fieldAnnotations.add(ApiParam.class);
+    fieldAnnotations.add(ApiImplicitParam.class);
```

Comment:
From the spec: "The @FormParam annotation is special and may only be utilized on resource and sub-resource methods. This is because it extracts information from a request entity. "
Please look: https://jersey.java.net/documentation/latest/jaxrs-resources.html#d0e2731
Also look old implementation: https://github.com/swagger-api/swagger-core/blob/399bb429803c03cff9aa474057c72df39026787f/modules/swagger-jaxrs/src/main/scala/com/wordnik/swagger/jaxrs/JaxrsApiReader.scala#L216-L219